datasets/klue/klue.py

```diff
@@ -0,0 +1,521 @@
+                "label": datasets.features.ClassLabel(names=["IT과학", "경제", "사회", "생활문화", "세계", "스포츠", "정치"]),
+                "url": datasets.Value("string"),
+                "date": datasets.Value("string"),
+            },
+            description=textwrap.dedent(
+                """\
+            In topic classification (TC), the goal is to predict the topic of a given text
+            snippet. We include TC in our KLUE benchmark, as inferring the topic of a text is a key
+            capability that should be possessed by a language understanding system.
+            Following a typical single  sentence classification task, we introduce YNAT, a Younhap
+            News Agency news headlines for Topic Classification. For Korean, no dataset has been
+            proposed for this task, which motivates us to construct the first Korean topic
+            classification benchmark. In this task, given a news headline, a text classifier must
+            predict a topic which is one of politics, economy, society, culture, world, IT/science,
+            and sports. Macro-F1 score is used to evaluate a system."""
+            ),
+            data_url=_DATA_URLs["ynat"],
+            url=_DESCRIPTION_URLs["ynat"],
+            file_map={
+                "train": "ynat-v1_train.json",
+                "dev": "ynat-v1_dev.json",
+            },
+        ),
+        KlueConfig(
+            name="sts",
+            features={
+                "guid": datasets.Value("string"),
+                "source": datasets.Value("string"),
+                "sentence1": datasets.Value("string"),
+                "sentence2": datasets.Value("string"),
+                "labels": {
+                    "label": datasets.Value("float64"),
+                    "real-label": datasets.Value("float64"),
+                    "binary-label": datasets.ClassLabel(names=["negative", "positive"]),
+                },
+            },
+            description=textwrap.dedent(
+                """\
+            STS is a task which aims to predict the semantic similarity of two input sentences as
+            a real value between 0 and 5. Note that we furthure binarized the prediction scores
+            into two classes with a threshold score 3.0 (paraphrased or not) and evaluated with
+            a classification metric.
+            """
+            ),
+            data_url=_DATA_URLs["sts"],
+            url=_DESCRIPTION_URLs["sts"],
+            file_map={
+                "train": "klue-sts-v1_train.json",
+                "dev": "klue-sts-v1_dev.json",
+            },
+        ),
+        KlueConfig(
+            name="nli",
+            features={
+                "guid": datasets.Value("string"),
+                "source": datasets.Value("string"),
+                "premise": datasets.Value("string"),
+                "hypothesis": datasets.Value("string"),
+                "label": datasets.ClassLabel(names=["entailment", "contradiction", "neutral"]),
```

Comment:
Could you use the same order for the labels as in the other NLI datasets ?

```suggestion
                "label": datasets.ClassLabel(names=["entailment", "neutral", "contradiction"]),
```